pkg/lib/authn/authenticator/service/service.go

```diff
@@ -555,7 +555,7 @@ func (s *Service) verifyWithSpec(info *authenticator.Info, spec *authenticator.S
 				channel = model.AuthenticatorOOBChannelSMS
 			}
 		}
-		kind := otp.KindOOBOTP(s.Config, channel)
+		kind := otp.KindOOBOTPCode(s.Config, channel)
```

Comment:
verifyWithSpec <- VerifyOneWithSpec <- NodeAuthenticateOOB
NodeAuthenticateOOB has OTPForm.
So we cannot use KindOOBOTPCode here.